### ISSUE TITLE
FITD Fails to decompile characters (3ds)

### ISSUE THREAD

**Issue by ih8ih8sn0w** [Sammi-Husky/Sm4sh-Tools#49]
When trying to decompile certain characters (Bayonetta for example), I get the following errors.
With the motion folder

```
C:\Users\ih8ih8sn0w\Desktop\SSB4\Tools\FITX_v0.77>FITD.exe C:\Users\ih8ih8sn0w\Desktop\SSB4\Tools\FITX_v0.77\bayonetta\motion.mtable -m C:\Users\ih8ih8sn0w\Desktop\SSB4\Tools\FITX_v0.77\bayonetta\body

> FITD v0.77 - Smash 4 Fighter Decompiler
> Licensed under the MIT License
> Copyright(c) 2016 Sammi Husky

>       Decompile ACMD.. -> "output\animcmd"

Unhandled Exception: System.ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
   at System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource)
   at SALT.Scripting.AnimCMD.ACMDScript.DeserializeLoop(Int32 startIndex, List`1& lines)
   at SALT.Scripting.AnimCMD.ACMDScript.DeserializeCommand(Int32 index, UInt32 ident, List`1& lines)
   at SALT.Scripting.AnimCMD.ACMDScript.Deserialize()
   at FITD.Program.write_movedef(ACMDScript game, ACMDScript effect, ACMDScript sound, ACMDScript expression, String animname, Boolean unlisted, String outdir)
   at FITD.Program.decompile_acmd(String mtable, String motionFolder, String output)
   at FITD.Program.Main(String[] args)
```

Without the motion folder

```
C:\Users\ih8ih8sn0w\Desktop\SSB4\Tools\FITX_v0.77>FITD.exe C:\Users\ih8ih8sn0w\Desktop\SSB4\Tools\FITX_v0.77\bayonetta\motion.mtable

> FITD v0.77 - Smash 4 Fighter Decompiler
> Licensed under the MIT License
> Copyright(c) 2016 Sammi Husky

>       Decompile ACMD.. -> "output\animcmd"

Unhandled Exception: System.ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
   at System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource)
   at SALT.Scripting.AnimCMD.ACMDScript.DeserializeLoop(Int32 startIndex, List`1& lines)
   at SALT.Scripting.AnimCMD.ACMDScript.DeserializeCommand(Int32 index, UInt32 ident, List`1& lines)
   at SALT.Scripting.AnimCMD.ACMDScript.Deserialize()
   at FITD.Program.write_movedef(ACMDScript game, ACMDScript effect, ACMDScript sound, ACMDScript expression, String animname, Boolean unlisted, String outdir)
   at FITD.Program.decompile_acmd(String mtable, String motionFolder, String output)
   at FITD.Program.Main(String[] args)
```

**Comment by Sammi-Husky** [comment]
Does this only happen on 3ds files or is it reproducible on Wii U files? I haven't been able to get this crash on WII U files

**Comment by Sammi-Husky** [comment]
Fixed with 615dcf7
